src/Pos/Communication/Tx.hs

```diff
@@ -71,6 +72,19 @@ submitMTx enqueue hdwSigner outputs = do
            createMTx utxo hdwSigner outputs
     submitAndSave enqueue txw
 
+-- | Construct Tx using secret key and given list of desired outputs without overriding TxDistr
+submitTxNoOverride
+    :: TxMode ssc ctx m
+    => EnqueueMsg m
+    -> SafeSigner
+    -> NonEmpty TxOutAux
+    -> m TxAux
+submitTxNoOverride enqueue ss outputs = do
```

Comment:
It's just for testing too. I don't think it's a good idea to mix working and testing control flows.

So I'll just document it.